### DISCUSSION TITLE
Configured inbound_emails but unable to receive them on Chatwoot when user reply back

### DISCUSSION THREAD

**Discussion by successful-fella** [chatwoot/chatwoot#1936] [Self hosted]
Hey all,

I had followed the [conversation continuity guide](https://www.chatwoot.com/docs/conversation-continuity/) and [email setup guide](https://www.chatwoot.com/docs/self-hosted/email-setup) to set up inbound emails. The emails when the user is offline goes to the user's email address but when a user replies back, the webhook is triggered (see logs) but the message won't show up on chatwoot.

Here are logs after the user replies to the email -
`srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.725386 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372] Started POST "/rails/action_mailbox/sendgrid/inbound_emails" for 167.89.116.31 at 2021-03-18 06:55:15 +0000
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.734964 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372] Processing by ActionMailbox::Ingresses::Sendgrid::InboundEmailsController#create as HTML
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.737242 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   Parameters: {"dkim"=>"none", "email"=>"Received: by mx0078p1las1.sendgrid.net with SMTP id d1ja4PmLfM Thu, 18 Mar 2021 06:55:14 +0000 (UTC)\nReceived: from forward501p.mail.yandex.net (unknown [77.88.28.111]) by mx0078p1las1.sendgrid.net (Postfix) with ESMTPS id DEAD22E0ADC for <reply+0365b30d-9961-4fae-8155-0080dbd7df77@app.kaypee.tech>; Thu, 18 Mar 2021 06:55:13 +0000 (UTC)\nReceived: from vla1-395bc165982e.qloud-c.yandex.net (vla1-395bc165982e.qloud-c.yandex.net [IPv6:2a02:6b8:c0d:179c:0:640:395b:c165]) by forward501p.mail.yandex.net (Yandex) with ESMTP id 49E0D3500299 for <reply+0365b30d-9961-4fae-8155-0080dbd7df77@app.kaypee.tech>; Thu, 18 Mar 2021 09:55:12 +0300 (MSK)\nReceived: from mail.yandex.com (mail.yandex.com [157.34.162.193]) by vla1-395bc165982e.qloud-c.yandex.net (mxback/Yandex) with HTTP id AtVr240dHa61-tBd4ra4r; Thu, 18 Mar 2021 09:55:11 +0300\nAuthentication-Results: vla1-395bc165982e.qloud-c.yandex.net; dkim=pass\nReceived: by vla1-937a7ab1b5ae.qloud-c.yandex.net with HTTP; Thu, 18 Mar 2021 09:55:11 +0300\nFrom: preetjuneja@zingalbox.com\nTo: Kawalpreet Juneja <reply+0365b30d-9961-4fae-8155-0080dbd7df77@app.kaypee.tech>\nIn-Reply-To: <conversation/0365b30d-9961-4fae-8155-0080dbd7df77/messages/23@app.kaypee.tech>\nReferences: <account/1/conversation/0365b30d-9961-4fae-8155-0080dbd7df77@app.kaypee.tech> <conversation/0365b30d-9961-4fae-8155-0080dbd7df77/messages/23@app.kaypee.tech>\nSubject: Re: [#1] New messages on this conversation\nMIME-Version: 1.0\nX-Mailer: Yamail [ http://yandex.ru ] 5.0\nDate: Thu, 18 Mar 2021 12:25:11 +0530\nMessage-Id: <292041616050386@mail.yandex.com>\nContent-Transfer-Encoding: base64\nContent-Type: text/html; charset=utf-8\n\nPGRpdj5UaGFuayB5b3UhPC9kaXY+PGRpdj7CoDwvZGl2PjxkaXY+MTguMDMu\nMjAyMSwgMTI6MjAsICJLYXdhbHByZWV0IEp1bmVqYSBmcm9tIExvY2FsIHRl\nc3RpbmciICZsdDttZUBrYXlwZWUudGVjaCZndDs6PC9kaXY+PGJsb2NrcXVv\ndGU+PGRpdiBiZ2NvbG9yPSIjZjZmNmY2IiBzdHlsZT0iYmFja2dyb3VuZC1j\nb2xvcjojZjZmNmY2O2JveC1zaXppbmc6Ym9yZGVyLWJveDtmb250LWZhbWls\neTonaGVsdmV0aWNhIG5ldWUnICwgJ2hlbHZldGljYScgLCAnYXJpYWwnICwg\nc2Fucy1zZXJpZjtmb250LXNpemU6MTRweDtoZWlnaHQ6MTAwJTtsaW5lLWhl\naWdodDoxLjZlbTttYXJnaW46MDt3aWR0aDoxMDAlICFpbXBvcnRhbnQiPjx0\nYWJsZSBiZ2NvbG9yPSIjZjZmNmY2IiBzdHlsZT0iYmFja2dyb3VuZC1jb2xv\ncjojZjZmNmY2O2JveC1zaXppbmc6Ym9yZGVyLWJveDtmb250LWZhbWlseTon\naGVsdmV0aWNhIG5ldWUnICwgJ2hlbHZldGljYScgLCAnYXJpYWwnICwgc2Fu\ncy1zZXJpZjtmb250LXNpemU6MTRweDttYXJnaW46MDt3aWR0aDoxMDAlIj48\ndGJvZHk+PHRyIHN0eWxlPSJib3gtc2l6aW5nOmJvcmRlci1ib3g7Zm9udC1m\nYW1pbHk6J2hlbHZldGljYSBuZXVlJyAsICdoZWx2ZXRpY2EnICwgJ2FyaWFs\nJyAsIHNhbnMtc2VyaWY7Zm9udC1zaXplOjE0cHg7bWFyZ2luOjAiPjx0ZCB2\nYWxpZ249InRvcCIgc3R5bGU9ImJveC1zaXppbmc6Ym9yZGVyLWJveDtmb250\nLWZhbWlseTonaGVsdmV0aWNhIG5ldWUnICwgJ2hlbHZldGljYScgLCAnYXJp\nYWwnICwgc2Fucy1zZXJpZjtmb250LXNpemU6MTRweDttYXJnaW46MDt2ZXJ0\naWNhbC1hbGlnbjp0b3AiPsKgPC90ZD48dGQgdmFsaWduPSJ0b3AiIHdpZHRo\nPSI2MDAiIHN0eWxlPSJib3gtc2l6aW5nOmJvcmRlci1ib3g7Y2xlYXI6Ym90\naCAhaW1wb3J0YW50O2ZvbnQtZmFtaWx5OidoZWx2ZXRpY2EgbmV1ZScgLCAn\naGVsdmV0aWNhJyAsICdhcmlhbCcgLCBzYW5zLXNlcmlmO2ZvbnQtc2l6ZTox\nNHB4O21hcmdpbjowIGF1dG8gMCBhdXRvO21heC13aWR0aDo2MDBweCAhaW1w\nb3J0YW50O3ZlcnRpY2FsLWFsaWduOnRvcCI+PGRpdiBzdHlsZT0iYm94LXNp\nemluZzpib3JkZXItYm94O2ZvbnQtZmFtaWx5OidoZWx2ZXRpY2EgbmV1ZScg\nLCAnaGVsdmV0aWNhJyAsICdhcmlhbCcgLCBzYW5zLXNlcmlmO2ZvbnQtc2l6\nZToxNHB4O21hcmdpbjowIGF1dG8gMCBhdXRvO21heC13aWR0aDo2MDBweDtw\nYWRkaW5nOjIwcHg7dGV4dC1hbGlnbjpjZW50ZXIiPjx0YWJsZSBiZ2NvbG9y\nPSIjZmZmIiBjZWxscGFkZGluZz0iMCIgY2VsbHNwYWNpbmc9IjAiIHdpZHRo\nPSIxMDAlIiBzdHlsZT0iYmFja2dyb3VuZC1jb2xvcjojZmZmO2JvcmRlcjox\ncHggc29saWQgI2U5ZTllOTtib3JkZXItcmFkaXVzOjRweDtib3gtc2l6aW5n\nOmJvcmRlci1ib3g7Zm9udC1mYW1pbHk6J2hlbHZldGljYSBuZXVlJyAsICdo\nZWx2ZXRpY2EnICwgJ2FyaWFsJyAsIHNhbnMtc2VyaWY7Zm9udC1zaXplOjE0\ncHg7bWFyZ2luOjA7dGV4dC1hbGlnbjpsZWZ0Ij48dGJvZHk+PHRyIHN0eWxl\nPSJib3gtc2l6aW5nOmJvcmRlci1ib3g7Zm9udC1mYW1pbHk6J2hlbHZldGlj\nYSBuZXVlJyAsICdoZWx2ZXRpY2EnICwgJ2FyaWFsJyAsIHNhbnMtc2VyaWY7\nZm9udC1zaXplOjE0cHg7bWFyZ2luOjAiPjx0ZCB2YWxpZ249InRvcCIgc3R5\nbGU9ImJveC1zaXppbmc6Ym9yZGVyLWJveDtmb250LWZhbWlseTonaGVsdmV0\naWNhIG5ldWUnICwgJ2hlbHZldGljYScgLCAnYXJpYWwnICwgc2Fucy1zZXJp\nZjtmb250LXNpemU6MTRweDttYXJnaW46MDtwYWRkaW5nOjIwcHg7dmVydGlj\nYWwtYWxpZ246dG9wIj48cD5IaSBrcCw8L3A+PHA+WW91IGhhdmUgbmV3IG1l\nc3NhZ2VzIG9uIHlvdXIgY29udmVyc2F0aW9uLjwvcD48dGFibGUgY2VsbHBh\nZGRpbmc9IjAiIGNlbGxzcGFjaW5nPSIwIiB3aWR0aD0iMTAwJSIgc3R5bGU9\nImJveC1zaXppbmc6Ym9yZGVyLWJveDtmb250LWZhbWlseTonaGVsdmV0aWNh\nIG5ldWUnICwgJ2hlbHZldGljYScgLCAnYXJpYWwnICwgc2Fucy1zZXJpZjtm\nb250LXNpemU6MTRweDttYXJnaW46MCI+PHRib2R5Pjx0cj48dGQ+PHN0cm9u\nZz5LYXdhbHByZWV0IEp1bmVqYTwvc3Ryb25nPjwvdGQ+PC90cj48dHI+PHRk\nPjxwPkhvcGVmdWxseTwvcD48L3RkPjwvdHI+PHRyPjx0ZD48c3Ryb25nPkth\nd2FscHJlZXQgSnVuZWphPC9zdHJvbmc+PC90ZD48L3RyPjx0cj48dGQ+PHA+\nSGV5PC9wPjwvdGQ+PC90cj48dHI+PHRkPjxzdHJvbmc+S2F3YWxwcmVldCBK\ndW5lamE8L3N0cm9uZz48L3RkPjwvdHI+PHRyPjx0ZD48cD5IZXk8L3A+PC90\nZD48L3RyPjx0cj48dGQ+PHN0cm9uZz5LYXdhbHByZWV0IEp1bmVqYTwvc3Ry\nb25nPjwvdGQ+PC90cj48dHI+PHRkPjxwPkFub3RoZXIgdGVzdDwvcD48L3Rk\nPjwvdHI+PHRyPjx0ZD48c3Ryb25nPkthd2FscHJlZXQgSnVuZWphPC9zdHJv\nbmc+PC90ZD48L3RyPjx0cj48dGQ+PHA+SGV5IHRlc3Q8L3A+PC90ZD48L3Ry\nPjx0cj48dGQ+PHN0cm9uZz5LYXdhbHByZWV0IEp1bmVqYTwvc3Ryb25nPjwv\ndGQ+PC90cj48dHI+PHRkPjxwPkhleTwvcD48L3RkPjwvdHI+PHRyPjx0ZD48\nc3Ryb25nPkthd2FscHJlZXQgSnVuZWphPC9zdHJvbmc+PC90ZD48L3RyPjx0\ncj48dGQ+PHA+VGVzdDwvcD48L3RkPjwvdHI+PHRyPjx0ZD48c3Ryb25nPkth\nd2FscHJlZXQgSnVuZWphPC9zdHJvbmc+PC90ZD48L3RyPjx0cj48dGQ+PHA+\nT2ZmbGluZSB0ZXN0PC9wPjwvdGQ+PC90cj48dHI+PHRkPjxzdHJvbmc+S2F3\nYWxwcmVldCBKdW5lamE8L3N0cm9uZz48L3RkPjwvdHI+PHRyPjx0ZD48cD5I\nZXkgYWdhaW48L3A+PC90ZD48L3RyPjx0cj48dGQ+PHN0cm9uZz5Zb3U8L3N0\ncm9uZz48L3RkPjwvdHI+PHRyPjx0ZD48cD5oaTwvcD48L3RkPjwvdHI+PHRy\nPjx0ZD48c3Ryb25nPkthd2FscHJlZXQgSnVuZWphPC9zdHJvbmc+PC90ZD48\nL3RyPjx0cj48dGQ+PHA+SGV5PC9wPjwvdGQ+PC90cj48L3Rib2R5PjwvdGFi\nbGU+PC90ZD48L3RyPjwvdGJvZHk+PC90YWJsZT48ZGl2IHN0eWxlPSJib3gt\nc2l6aW5nOmJvcmRlci1ib3g7Y2xlYXI6Ym90aDtjb2xvcjojOTk5O2ZvbnQt\nZmFtaWx5OidoZWx2ZXRpY2EgbmV1ZScgLCAnaGVsdmV0aWNhJyAsICdhcmlh\nbCcgLCBzYW5zLXNlcmlmO2ZvbnQtc2l6ZToxNHB4O21hcmdpbjowO3BhZGRp\nbmc6MjBweDt3aWR0aDoxMDAlIj48dGFibGUgd2lkdGg9IjEwMCUiIHN0eWxl\nPSJib3gtc2l6aW5nOmJvcmRlci1ib3g7Zm9udC1mYW1pbHk6J2hlbHZldGlj\nYSBuZXVlJyAsICdoZWx2ZXRpY2EnICwgJ2FyaWFsJyAsIHNhbnMtc2VyaWY7\nZm9udC1zaXplOjE0cHg7bWFyZ2luOjAiPjx0Ym9keT48dHIgc3R5bGU9ImJv\neC1zaXppbmc6Ym9yZGVyLWJveDtmb250LWZhbWlseTonaGVsdmV0aWNhIG5l\ndWUnICwgJ2hlbHZldGljYScgLCAnYXJpYWwnICwgc2Fucy1zZXJpZjtmb250\nLXNpemU6MTRweDttYXJnaW46MCI+PC90cj48dHIgc3R5bGU9ImJveC1zaXpp\nbmc6Ym9yZGVyLWJveDtmb250LWZhbWlseTonaGVsdmV0aWNhIG5ldWUnICwg\nJ2hlbHZldGljYScgLCAnYXJpYWwnICwgc2Fucy1zZXJpZjtmb250LXNpemU6\nMTRweDttYXJnaW46MCI+PHRkIHZhbGlnbj0idG9wIiBzdHlsZT0iYm94LXNp\nemluZzpib3JkZXItYm94O2ZvbnQtZmFtaWx5OidoZWx2ZXRpY2EgbmV1ZScg\nLCAnaGVsdmV0aWNhJyAsICdhcmlhbCcgLCBzYW5zLXNlcmlmO2ZvbnQtc2l6\nZToxNHB4O21hcmdpbjowO3BhZGRpbmc6MCAwIDIwcHggMDt2ZXJ0aWNhbC1h\nbGlnbjp0b3AiPlBvd2VyZWQgYnkgPGEgaHJlZj0iaHR0cHM6Ly93d3cuY2hh\ndHdvb3QuY29tLyIgcmVsPSJub29wZW5lciBub3JlZmVycmVyIiBzdHlsZT0i\nYm94LXNpemluZzpib3JkZXItYm94O2NvbG9yOiM5OTk7Zm9udC1mYW1pbHk6\nJ2hlbHZldGljYSBuZXVlJyAsICdoZWx2ZXRpY2EnICwgJ2FyaWFsJyAsIHNh\nbnMtc2VyaWY7Zm9udC1zaXplOjE0cHg7bWFyZ2luOjA7cGFkZGluZzowIDAg\nMjBweCAwO3RleHQtYWxpZ246Y2VudGVyO3ZlcnRpY2FsLWFsaWduOnRvcCI+\nIENoYXR3b290IDwvYT48L3RkPjwvdHI+PC90Ym9keT48L3RhYmxlPjwvZGl2\nPjwvZGl2PjwvdGQ+PHRkIHZhbGlnbj0idG9wIiBzdHlsZT0iYm94LXNpemlu\nZzpib3JkZXItYm94O2ZvbnQtZmFtaWx5OidoZWx2ZXRpY2EgbmV1ZScgLCAn\naGVsdmV0aWNhJyAsICdhcmlhbCcgLCBzYW5zLXNlcmlmO2ZvbnQtc2l6ZTox\nNHB4O21hcmdpbjowO3ZlcnRpY2FsLWFsaWduOnRvcCI+wqA8L3RkPjwvdHI+\nPC90Ym9keT48L3RhYmxlPjwvZGl2PjwvYmxvY2txdW90ZT48ZGl2PsKgPC9k\naXY+\n", "to"=>"Kawalpreet Juneja <reply+0365b30d-9961-4fae-8155-0080dbd7df77@app.kaypee.tech>", "from"=>"preetjuneja@zingalbox.com", "sender_ip"=>"77.88.28.111", "envelope"=>"{\"to\":[\"reply+0365b30d-9961-4fae-8155-0080dbd7df77@app.kaypee.tech\"],\"from\":\"preetjuneja@zingalbox.com\"}", "subject"=>"Re: [#1] New messages on this conversation", "charsets"=>"{\"to\":\"UTF-8\",\"subject\":\"UTF-8\",\"from\":\"UTF-8\"}", "SPF"=>"pass"}
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.818838 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]    (0.4ms)  BEGIN
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.824800 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   ActionMailbox::InboundEmail Create (4.2ms)  INSERT INTO "action_mailbox_inbound_emails" ("message_id", "message_checksum", "created_at", "updated_at") VALUES ($1, $2, $3, $4) RETURNING "id"  [["message_id", "292041616050386@mail.yandex.com"], ["message_checksum", "685b51968c522ecfb664c62544faf4c1f69469f4"], ["created_at", "2021-03-18 06:55:15.810470"], ["updated_at", "2021-03-18 06:55:15.810470"]]
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.838423 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   ActiveStorage::Blob Load (2.4ms)  SELECT "active_storage_blobs".* FROM "active_storage_blobs" INNER JOIN "active_storage_attachments" ON "active_storage_blobs"."id" = "active_storage_attachments"."blob_id" WHERE "active_storage_attachments"."record_id" = $1 AND "active_storage_attachments"."record_type" = $2 AND "active_storage_attachments"."name" = $3 LIMIT $4  [["record_id", 9], ["record_type", "ActionMailbox::InboundEmail"], ["name", "raw_email"], ["LIMIT", 1]]
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.854245 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   ActiveStorage::Attachment Load (0.6ms)  SELECT "active_storage_attachments".* FROM "active_storage_attachments" WHERE "active_storage_attachments"."record_id" = $1 AND "active_storage_attachments"."record_type" = $2 AND "active_storage_attachments"."name" = $3 LIMIT $4  [["record_id", 9], ["record_type", "ActionMailbox::InboundEmail"], ["name", "raw_email"], ["LIMIT", 1]]
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.864709 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   ActiveStorage::Blob Create (1.4ms)  INSERT INTO "active_storage_blobs" ("key", "filename", "content_type", "metadata", "byte_size", "checksum", "created_at") VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING "id"  [["key", "znk2tdlm8m8xc1778q1rkr614pi8"], ["filename", "message.eml"], ["content_type", "message/rfc822"], ["metadata", "{\"identified\":true}"], ["byte_size", 7445], ["checksum", "VaYe8/ByyyO8tn11Zjyo1w=="], ["created_at", "2021-03-18 06:55:15.859849"]]
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.869360 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   ActiveStorage::Attachment Create (1.6ms)  INSERT INTO "active_storage_attachments" ("name", "record_type", "record_id", "blob_id", "created_at") VALUES ($1, $2, $3, $4, $5) RETURNING "id"  [["name", "raw_email"], ["record_type", "ActionMailbox::InboundEmail"], ["record_id", 9], ["blob_id", 9], ["created_at", "2021-03-18 06:55:15.866994"]]
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.878933 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   ActionMailbox::InboundEmail Update (0.8ms)  UPDATE "action_mailbox_inbound_emails" SET "updated_at" = $1 WHERE "action_mailbox_inbound_emails"."id" = $2  [["updated_at", "2021-03-18 06:55:15.872027"], ["id", 9]]
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | D, [2021-03-18T06:55:15.882749 #1] DEBUG -- : [92f46427-91dc-4a80-ba28-510e49f9f372]    (2.3ms)  COMMIT
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.897545 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372]   Disk Storage (8.6ms) Uploaded file to key: znk2tdlm8m8xc1778q1rkr614pi8 (checksum: VaYe8/ByyyO8tn11Zjyo1w==)
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.915603 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372] [ActiveJob] Enqueued ActionMailbox::RoutingJob (Job ID: b643bfea-82db-40cf-9069-12a21207c08a) to Sidekiq(action_mailbox_routing) with arguments: #<GlobalID:0x00007f74cbf14e00 @uri=#<URI::GID gid://chatwoot/ActionMailbox::InboundEmail/9>>
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.923496 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372] [ActiveJob] Enqueued ActiveStorage::AnalyzeJob (Job ID: d6b4418b-687f-4849-afc7-edd817ab22c9) to Sidekiq(active_storage_analysis) with arguments: #<GlobalID:0x00007f74cc38ee90 @uri=#<URI::GID gid://chatwoot/ActiveStorage::Blob/9>>
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.925450 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372] No template found for ActionMailbox::Ingresses::Sendgrid::InboundEmailsController#create, rendering head :no_content
srv-captain--chatwoot-web.1.fhum9aflc2wd@caprover    | I, [2021-03-18T06:55:15.927618 #1]  INFO -- : [92f46427-91dc-4a80-ba28-510e49f9f372] Completed 204 No Content in 188ms (ActiveRecord: 13.7ms | Allocations: 4674)
`

**Comment by sojan-official** [comment]
@successful-fella  can you check and see if you have `MAILER_INBOUND_EMAIL_DOMAIN` environment variable configured?

ref: https://github.com/chatwoot/chatwoot/issues/1450

**Reply by successful-fella** [reply]
It wasn't. I had set it up now with value as same as root domain in sendgrid, is that what its value supposed to be? I'm still not receiving the replies on chatwoot though the webhook triggers like the above logs.

**Reply by JacoBezuidenhout** [reply]
Hello @successful-fella .... did you manage to figure this out? I am having the same challenge...

**Reply by JacoBezuidenhout** [reply]
Found the issue... I had to ensure that SMTP_DOMAIN is the same as the MAILER_INBOUND_EMAIL_DOMAIN. Then it worked.
